tests/integration/network_test.py

```diff
@@ -7,7 +7,6 @@
 from ..base import requires_api_version
 
 
-@requires_api_version('1.21')
```

Comment:
I wasn't sure if the two would interact weirdly, so I elected to be more explicit, but I don't have strong feelings about it